Team3 individual skills section

liux2 opened the pull request:
This Pull Request adds the content needed for the individual section of the Assessment Guide.

reid7danny committed:
Wrote assessment descriptions for Understanding and avoiding red flags

reid7danny committed:
Added formatting to Individual Skills

reid7danny committed:
Fixed blank lines that broke Build

reid7danny committed:
Wrote the section for Continuous Learning

reid7danny committed:
Fixed line lengths causing build to break

reid7danny committed:
Line length change

reid7danny committed:
Reformatted Individual Skills and Added to Understanding Best Practices

Commit:
Delete Mac Os self created file

Commit:
created nested bullet points to create levels

Commit:
Implemented changes from assessment issue trackers.

Commit:
Implemented more changes from issue trackers in README FILE.

Commit:
Fixed build issue with line length.

lussierc commented:
This looks good. Thanks for adding the changes suggested by @Jordan-A in Issue #15!

lussierc committed:
Remove periods at end of lines in Individual Skills section.

This was done to adhere to set style guide in README and in course Markdown repo.

lussierc committed:
Update with small formatting changes to Individual Skills section.

enpuyou commented on README.md:
```diff
@@ -64,14 +64,14 @@ members would not write Python source code that also manifests this red flag.
+  * :disappointed: = `pyenv` and `pipenv` not installed or setup on laptop
+  * :confused: = Unable to run programs with `pyenv` and `pipenv`
+  * :neutral_face: = Able to run programs with `pyenv` and `pipenv`
+  * :smiley: = Able to run programs with `pyenv` and `pipenv` using additional commands
```
Can you be more specific about what the "additional commands and flags" actually are? Other students might be confused about what "additional commands" they should run in order to achieve the level of  :sunglasses:

lussierc replied:
@enpuyou Hi Enpu. This content was already included originally in the repository by the instructor. The only thing we changed was the letter criteria system to the emoji's, whereas the text is still from the course instructor.

As a result, I don't think it is necessary to change this text.

enpuyou reviewed (changes requested):
There is an inconsistency in the use of "Excellent", "E", and what your team has previously used in other sections of assessment <https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment/pull/7>

> > The content looks great! One more point that might worth discussing:
> > Your team added an equal sign after each emoji and a further explanation on the highest level requirement of each assessment while a couple of other teams have not. Should there be a more specific standard for the assessment then?
> 
> @enpuyou We followed the Markdown standard that all teams should be following with regards to the emoji's and `=` signs. This standard can be found in [this file](https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown/blob/master/AssessmentFormat.md) which is apart of the Markdown repository for the course. All teams should be following this standard as far as I know. With regards to the higher level explanation, this appeared in the README examples in the Assessment repository and the previously mentioned Markdown standard file also mentions that a detailed paragraph describing the subcategory should be included.

enpuyou committed:
Merge branch 'master' into team3-individual-skills-section

lussierc commented:
> There is an inconsistency in the use of "Excellent", "E", and what your team has previously used in other sections of assessment #7
> 
> > > The content looks great! One more point that might worth discussing:
> > > Your team added an equal sign after each emoji and a further explanation on the highest level requirement of each assessment while a couple of other teams have not. Should there be a more specific standard for the assessment then?
> > 
> > 
> > @enpuyou We followed the Markdown standard that all teams should be following with regards to the emoji's and `=` signs. This standard can be found in [this file](https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown/blob/master/AssessmentFormat.md) which is apart of the Markdown repository for the course. All teams should be following this standard as far as I know. With regards to the higher level explanation, this appeared in the README examples in the Assessment repository and the previously mentioned Markdown standard file also mentions that a detailed paragraph describing the subcategory should be included.

Hi @enpuyou, I was wondering where this was specifically used? I cannot seem to find this issue.

enpuyou commented on README.md:
```diff
@@ -32,11 +32,11 @@ professional skills on which you want to be assessed during the Spring 2020
+* :sunglasses: = Excellent
 
 For each skill and its assessment levels, you should also clearly explain what a
 student must do to demonstrate mastery at the level of "Excellent". You should
```
Here

lussierc replied:
Thanks, I fixed it @enpuyou.

enpuyou commented on README.md:
```diff
@@ -48,11 +48,11 @@ finished with the assessment, please update the README as necessary.
+  * :sunglasses: = Identifies, understands, and fixes all red flags, while also providing
     documentation, tools, and advice that help team members develop this skill
 
 A software engineer demonstrates the mastery of this professional skill at the E
```
Here

lussierc replied:
Ah, thank you @enpuyou.

lussierc replied:
Thanks, I fixed it @enpuyou.

enpuyou commented on README.md:
```diff
@@ -64,14 +64,14 @@ members would not write Python source code that also manifests this red flag.
+    commands and flags to streamline workflow, while being able to provide
+    assistance and troubleshooting to other students using `pyenv` and `pipenv`
 
 A software engineer demonstrates the mastery of this technical skill at the E
```
Here

lussierc replied:
Thanks, I fixed it @enpuyou.

lussierc committed:
Add emojis and remove old criteria assessment system in README.

enpuyou committed:
Merge branch 'master' into team3-individual-skills-section

enpuyou committed:
Merge branch 'master' into team3-individual-skills-section

enpuyou committed:
Merge branch 'master' into team3-individual-skills-section

enpuyou committed:
Merge branch 'master' into team3-individual-skills-section

enpuyou commented:
@liux2 @lussierc I'd like to have two more members from your group to review this PR and approve if it's believed to be up to standard

lussierc reviewed (approved):
I am approving this PR. The PR accurately updates the README with the new emoji scale. Additionally, it also offers what seem to be reasonably descriptive and informative content for the Individual Skills section, without noticeable issues. I also will note that Team 3 has taken steps to make improvements based on the Technical Leaders suggestions.

enpuyou commented on assessment.md:
```diff
@@ -130,9 +130,50 @@
+    practices,
+    such as labeling commits, working efficiently and
+    utilizing resources to assist in problem solving
+  * 😕 = Adopts a few good practices, but does not
```
As I am looking at this again, there is another change needs to be made before merging. The use of emoji here is incorrect as it should be in the format like this `:disappointed:` instead of "😞". It should be good to go after this issue is resolved. I'm sorry for this going back and forth!

lussierc replied:
@enpuyou, interesting, I see what you mean. I just changed them!

lussierc committed:
Add Markdown emoji codes to replace actual emoji icons in Individual Skills.
